README.md

```diff
@@ -1,2 +1,52 @@
-# pcb
-Monolithic repository containing QFSAE PCB designs.
+# QFSAE PCB Projects
+
+This repository stores the various PCB designs made by the QFSAE team as well as useful reference designs.
+
+# Altium
+
+The QFSAE team designs its boards using Altium. If you do not have Altium installed, Please contact Logan or Brent to obtain the appropriate licensing information to get Altium up and running on your computer. In regards to manufacturing, the team most often makes use of [JLCPCB](https://jlcpcb.com/). Traditionally, PCBs made by QFSAE members have been populated by hand in house. However, due to the restrictions imposed on team operation by the ongoing COVID-19 pandemic, it has become of utmost importance to create an Altium workflow which allows us to outsource our board assembly in an efficient manner. JLCPCB offers an economical assembly service for surface mount components with the following caveats.
+
+- All surface mount components must be on one side of the board
+- Through hole parts will not be soldered
+- The components must be from the JLCPCB parts that they keep in stock at their manufacturing facility.
+- The part being populated on the board must be within the size and clearance constraints given by JLC PCB.
+
+In order to meet these manufacturing conditions, we must restrain the SMD parts we use to those offered by JLCPCB and use Altium design rules that reflect the manufacturing capabilities of JLCPCB when checking our PCB layouts. The parts library is quite vast with over 30,000 components. With that being said, there will still be cases where a custom footprint or one from another library must be used. In that case, check with another team member to verify that it can be hand soldered properly. The sections below provide instructions on importing the JLC parts libraries, which are included in this repository as well as design rules.
+
+## Library Setup
+TODO
+
+## Design Rule Configuration
+
+TODO 
+
+## Version Control
+
+Altium has integrated Git support. The instructions below show how to configure the Git extension. However, using command line git or some other client is also fine. 
+
+TODO
+
+# Design Review Checklist
```

Comment:
Some suggestions:
- Dimensions check
- Nothing obstructed on silkscreen; component labels make sense
- No missing traces or disconnected grounds, islands removed
- Proper trace width adjustments where applicable (wider for power rails etc.)
- No blind vias on 4 layer boards
- I/O ports and headers are placed in sensible locations

It might be a good idea to split the checklist by characteristic (ie. general, 2 layer, 4 layer) as not every item will apply across the board.

Reply:
These have been incorporated into the checklists.